### DISCUSSION TITLE
倒排索引添加自定义分词

### DISCUSSION THREAD

**Discussion by gospider001** [apache/doris#33285] [Ideas]
倒排索引 的构建全是基于内置的分词的，有没有办法自定义分词，这样就完美了

**Comment by yagagagaga** [comment]
This will be supported in the Q3.

**Comment by ClareChengHY** [comment]
这个自定义分词有支持了吗，我在官方的文档中仍然没有看到相关的描述

**Comment by echo-hhj** [comment]
you can refer this: https://doris-forum.org.cn/questions/D1S71/doris2-0-3-dao-pai-suo-yin-de-match-any-jie-guo-yu-wen-dang-bu-fu
